store/src/java/com/zimbra/cs/mailbox/Mailbox.java

```diff
@@ -10649,6 +10655,7 @@ public void setSavedSearchPromptStatus(OperationContext octxt, String searchStri
     public static class MessageCallbackContext {
```

Comment:
Why is this class inside Mailbox.java? If there is no need to share class members, I would prefer to move int to it's own file, because Mailbox.java is enormous as it is.

Reply:
Yeah good point. I'll move it into its own file.